package.json

```diff
@@ -87,19 +89,24 @@
   },
   "pre-commit": "lint:staged",
   "dependencies": {
+    "@babel/preset-typescript": "^7.18.6",
     "@emotion/react": "^11.10.5",
     "@emotion/styled": "^11.10.5",
     "@mui/icons-material": "^5.11.0",
     "@mui/material": "^5.11.5",
     "@mui/styled-engine": "^5.11.0",
     "@mui/styled-engine-sc": "^5.11.0",
+    "@storybook/addon-controls": "^6.5.16",
+    "@storybook/addons": "^6.5.16",
     "apisauce": "2.1.1",
     "chalk": "4.1.1",
     "compression": "1.7.4",
+    "concurrently": "^7.6.0",
     "cross-env": "7.0.3",
     "dotenv": "10.0.0",
     "express": "4.16.4",
     "history": "4.9.x",
+    "http-server": "^14.1.1",
```

Comment:
all the new ones that you have added here are dev dependencies. Can we move it to dev dependencies section ?